package.json

```diff
@@ -4,15 +4,16 @@
   "description": "React-fiber renderer for THREE.js",
   "main": "dist/index.cjs.js",
   "module": "dist/index.js",
+  "types": "types/index.d.ts",
```

Comment:
here is where we reference the entry point for the types file, which other npm projects will be able to understand: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html